doc/y_cable/design_doc.md

```diff
@@ -0,0 +1,226 @@
+## Multiple y cable vendor API design
```

Comment:
s/y cable/Y-cable/

Please do this throughout the document

Reply:
fixed

Reply:
I don't see any difference.

Reply:
fixed now